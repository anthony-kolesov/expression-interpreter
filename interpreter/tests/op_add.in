# Typical operation.
out 5 + 3
print "\n"

# Relatively big numbers.
out 1000000 + 1
print "\n"

# Crossing the zero.
out -9 + 12
print "\n"

# Staying negative.
out -9 + 5 
print "\n"

# Floating point numbers.
out 3.4 + 1.566
print "\n"

# Floating point numbers crossing zero.
out -72.5 + 91
print "\n"

# Zero values.
out 4 + 0
print "\n"

out 0 + 6
print "\n"

out 0 + 0
print "\n"

# Value 1.
out 1 + 7
print "\n"

out 7 + -1
print "\n"
