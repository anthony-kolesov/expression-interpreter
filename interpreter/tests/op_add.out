8
1000001
3
-4
4.966000
18.500000
4
6
0
8
6
